data/controls.xml

```diff
@@ -305,7 +306,7 @@
         <label fallback="label">label</label>
     </control>
     <control type="rss" id="int">
-        <width>int</width>
+        <width min="int" max="int">int</width>
```

Comment:
hmm "min" and "max" do not make too much sense to me for a RSS control. Did you test if this has any influence?